src/js/media-controller.js

```diff
@@ -551,6 +552,17 @@ class MediaController extends MediaContainer {
     this.enableHotkeys();
   }
 
+  get fullscreenElement() {
+    if (this.hasAttribute('fullscreen-element')) {
+      return document.getElementById(this.getAttribute('fullscreen-element'));
+    }
+    return this.#fullscreenElement ?? this;
+  }
+
+  set fullscreenElement(element) {
+    this.#fullscreenElement = element;
+  }
```

Comment:
I would assume that if this property is set, it should unset the attribute/getter should check the private var before checking the attribute.

Reply:
Yeah I held off on that for a couple of reasons:

1. I was trying to avoid @luwes's common "attribute sprouting" concerns here.
2. It wasn't obvious to me that if someone sets and then unsets the value that they'd want the `fullscreen-element` attribute to "disappear into the aether"

Originally I just had an `attributeChangedCallback` case that would explicitly set the `fullscreenElement`, but I ended up removing that to allow for the separation and precendence of the different values that I implemented here.

That said, I don't feel strongly about this impl, and it's easy enough to change. If we want to treat them as "synonymous", I think we would need to re-add my `attributeChangedCallback` logic and do what you're proposing.